lib/jobs/ServiceInstanceUpdateJob.js

```diff
@@ -150,7 +150,7 @@ class ServiceInstanceUpdateJob extends BaseJob {
       const jobData = _.cloneDeep(instanceDetails);
       jobData.attempt = jobData.attempt + 1;
       if (trackAttempts) {
-        if (jobData.attempt > config.scheduler.jobs.service_instance_update.max_attempts) {
+        if (jobData.attempt > _.get(instanceDetails, 'max_attempts', config.scheduler.jobs.service_instance_update.max_attempts)) {
           logger.error(`Auto udpate for instance ${instanceDetails.instance_id}  has exceeded max configured attempts : ${config.scheduler.jobs.service_instance_update.max_attempts}}`);
```

Comment:
same as above , print correct value in log for max_attempts and reschedule_delay.